App/xcbuddykit/Sources/Commands/InitCommand.swift

```diff
@@ -27,6 +33,11 @@ enum InitCommandError: FatalError {
 
 /// Command that initializes a Project.swift in the current folder.
 public class InitCommand: NSObject, Command {
+    
+    enum WizardQuestion {
+        static let name = "1. What's the name of your project? (Leave empty to use the name of the project folder: %@)"
```

Comment:
I think it might be too long for the terminal. What about?

```bash
1. What's the name to the project? (default: XXX)
2. In which directory? (default: YYY)
```
Where XXX and YYY are the current directory name and path respectively. What do you think?

Reply:
Yes, agree. Anyway, those questions are temporary, we should check which is the best copy to ask for project name, path, etc